src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -99,16 +99,22 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             throw new RuntimeException('Either a "route_name" or a "method" operation attribute must exist.');
         }
 
+        $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
```

Comment:
We can simplify it like this:

```php
$actionName = sprintf('%s_%s', strtolower($operationName), $collection ? 'collection' : 'item');
$controller = $operation['controller'] ?? self::DEFAULT_ACTION_PATTERN.$actionName;
```

This means if you set a custom operation name that's different from the method, it'll not match the default action. But I think that's not unreasonable.